package.json

```
@@ -91,6 +92,7 @@
     "jest-watch-select-projects": "^2.0.0",
     "jest-watch-typeahead": "^0.6.5",
     "jsonfile": "^6.0.1",
+    "lodash": "^4.17.21",
```

Could we use `lodash.camelcase` instead?

https://www.npmjs.com/package/lodash.camelcase

```suggestion
    "lodash.camelcase": "^4.3.0",
```